docs/guides/installation.md

```diff
@@ -24,10 +24,174 @@ Installation requires the following:
 Clone the [ZARP workflow repository][zarp] with:
 
 ```sh
-git clone git@github.com:zavolanlab/zarp
-# or: git clone https://github.com/zavolanlab/zarp.git
+git clone https://github.com/zavolanlab/zarp.git
 ```
 
 ### 2. Set up Conda environment
 
-**TODO:** Add on...
+To check if you already have conda installed in your system type:
+
+```sh
+conda --version
+```
+
+Example:
+```sh
+conda --version
+conda 22.11.1
+```
+If it is not installed, you will see a <code style="color: red;">command not found error.</code>
+
+### Conda installation
```

Comment:
Check the headers, this should probably not be on the same level as the numbered headers, else the navigation on the side will be messed up.

A markdown linter would help here and make the overall doc look cleaner.

Also, the installation of Conda is complex (multiple flavors, miniconda vs miniforge etc.) and changes frequently. Maybe it's better to just link to the official docs and make it their job to keep it up to date?

Reply:
I added a sentence that recommends the official documentation of Miniconda but left the instructions that we have.